proxy/src/telemetry/metrics.rs

```diff
@@ -81,6 +81,9 @@ enum End {
 
 #[derive(Debug, Default)]
 struct TransportStats {
+    // TODO: we may want to have more distinction between transports, if
+    // we want to collect different metrics based on transport protocol?
```

Comment:
What is this TODO referencing, exactly? Is it worth being a discussion separately in an issue?